### DISCUSSION TITLE
Question on interface type 2 with spotflag=20,21 or 22

### DISCUSSION THREAD

**Discussion by halfwitbm** [OpenRadioss/OpenRadioss#1940] [Q&A] [answered]
Hi All,

When I try to use Radioss interface type 2 with spotflag=20,21,22 to realize two surface tie-break test, I find It is hard to set parameter on type 2 option card for spotflag=20,21,22. I searched Radioss example web, only find the example "RD-E: 4500 Multi-Domain" has interface type 2, but it didn't use tie-fail mechanism. Can someone give me a Radioss example Input Deck which use this tie-fail mechanism?  or help me make a very simple interface type 2 case: At begining two small surface tied by interface type 2 with spotflag=20,21,22, one surface fixed position all the time, other surface has a displacement then this connecting spotweld failed. I tried to create it on Radioss, but always failed to run.

Thank you.

Bin

**Comment by MarianBulla** [comment, marked as answer]
Hi Bin,
Please find attached a simple example with the tied_with_failure interface TYPE2:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/0261757e-853c-4132-952e-3100de78f802)
[2023_Tied_Interface_Type2_with_Failure.zip](https://github.com/OpenRadioss/OpenRadioss/files/12806560/2023_Tied_Interface_Type2_with_Failure.zip)

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by halfwitbm** [reply]
Hi Marian,

Thank you very much, Marian. It is a great example for me to understand interface type 2 card. I am really appreciated.

Best Regards,

Bin
